docker/docker-compose.yml

```diff
@@ -284,8 +284,8 @@ services:
     container_name: s3
     image: minio/minio
     volumes:
-      - ~/minio/data:/data
-      - ~/.minio/certs:/certs
+      - ./dev/minio/data:/data
+      - ./dev/minio/certs:/certs
```

Comment:
Having moved to HTTPR these relevant anymore?

Reply:
Yeah, that's just to cover the bases, you can still use https if you want.